packages/app/src/debug/DebugSpec.cy.tsx

```diff
@@ -0,0 +1,96 @@
+import type { Spec, TestResults } from './DebugSpec.vue'
+import DebugSpec from './DebugSpec.vue'
+
+describe('<DebugSpec/> with multiple test results', () => {
+  const spec: Spec = {
+    id: '8879798756s88d',
+    path: 'cypress/tests/auth.spec.ts',
+  }
+
+  const testResults: TestResults[] = [
+    {
+      id: '676df87878',
+      titleParts: ['Login', 'Should redirect unauthenticated user to signin page'],
+    },
+    {
+      id: '78hjkdf987d9f',
+      titleParts: ['Login', 'redirects to stored path after login'],
+    },
+  ]
+
+  it('mounts correctly', () => {
+    cy.mount(() => (
+      <DebugSpec spec={spec} testResults={testResults} />
+    ))
+
+    cy.findByTestId('debug-spec-item').children().should('have.length', 3)
+    cy.findByTestId('spec-contents').children().should('have.length', 2)
+    cy.findByTestId('spec-path').should('have.text', 'cypress/tests/auth.spec.ts')
+    cy.contains('auth').should('have.css', 'color', 'rgb(67, 72, 97)')
+    cy.findByTestId('run-failures').should('not.be.disabled')
+    .should('have.text', ' Run Failures ')
+    .should('have.css', 'color', 'rgb(73, 86, 227)')
```

Comment:
I am not sure we should assert CSS like this - I think the snapshot with percy should cover this.

Reply:
I agree. The CSS assertions have been removed on the latest commit